lib/guardian.ex

```diff
@@ -29,7 +29,7 @@ defmodule Guardian do
   Any other option provided will be merged with the configuration in the config
   files.
 
-  It contains some generated functions, and some callbacks
+  The Guardian module contains some generated functions and some callbacks.
```

Comment:
I believe that the comma goes there.

Don't quote me on it because I am not a native English speaker.

But I learned from Grammarly that it is good to add that comma there so you know it is part of the listing rather than grouping.

For example A and B,  C, and D

Replace A, B, C, and D with something that makes sense.

Reply:
This change is fine 👍